boards/arm/stm32l476g_disco/doc/stm32l476g_disco.rst

```diff
@@ -0,0 +1,231 @@
+.. _stm32l476g_disco_board:
+
+ST STM32L476G Discovery
+########################
+
+Overview
+********
+
+The STM32L476G Discovery board features an ARM Cortex-M4 based STM32L476VG MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the STM32L476G Discovery board:
+
+
+- STM32L476VGT6 microcontroller featuring 1 Mbyte of Flash memory, 128 Kbytes of RAM in LQFP100 package
+- On-board ST-LINK/V2-1 supporting USB re-enumeration capability
+- Three different interfaces supported on USB:
+
+        - Virtual com port
+        - Mass storage
+        - Debug port
+
+- LCD 24 segments, 4 commons in DIP 28 package
+- Seven LEDs:
+
+        - LD1 (red/green) for USB communication
+        - LD2 (red) for 3.3 V power on
+        - LD3 Over current (red)
+        - LD4 (red), LD5 (green) two user LEDs
+        - LD6 (green), LD7 (red) USB OTG FS LEDs
+
+- Pushbutton (reset)
+- Four directions Joystick with selection
+- USB OTG FS with micro-AB connector
+- SAI Audio DAC, Stereo with output jack
+- Digital Microphone MEMS Accelerometer and magnetometer Mems
+- Gyroscope MEMS
+- 128-Mbit Quad-SPI Flash memory
+- MCU current ammeter with 4 ranges and auto calibration
+- Connector for external board or RF-EEPROM
+- Four power supply options:
+- ST-LINK/V2-1
```

Comment:
This list should be indented (they're the four power supply options, right)?  and there should be four listed instead of three.